src/shared/telemetry/activation.ts

```diff
@@ -72,24 +75,28 @@ export async function activate(activateArguments: {
         activateArguments.extensionContext.subscriptions
     )
 }
-
-export function isTelemetryEnabled(toolkitSettings: SettingsConfiguration): boolean {
+/*
+ * Formats the AWS telemetry setting to a boolean: false if setting value was 'Disable' or false, true for everything else
+ */
+export async function formatTelemetrySettingToBool(toolkitSettings: SettingsConfiguration): Promise<void> {
```

Comment:
Consider a less verbose but more concise name like `sanitizeTelemetrySetting`

Reply:
Changed to your suggestion.